src/events/deposits.upsert.service.ts

```diff
@@ -255,4 +257,37 @@ export class DepositsUpsertService {
       );
     });
   }
+
+  async syncDepositedIron(): Promise<void> {
+    const aggregate = await this.prisma.deposit.aggregate({
+      _sum: {
+        amount: true,
+      },
+    });
+    this.influxDbService.writePoints([
+      {
+        measurement: 'deposited_iron',
+        fields: [
+          {
+            name: 'iron',
+            type: 'float',
+            value: aggregate._sum.amount || 0,
+          },
+        ],
+        tags: [],
+        timestamp: new Date(),
+      },
+    ]);
+    const runAt = new Date();
+    runAt.setMinutes(runAt.getMinutes() + 10);
+    await this.graphileWorkerService.addJob(
+      GraphileWorkerPattern.SYNC_DEPOSITED_IRON,
+      {},
+      {
```

Comment:
Do I need to define this queue somewhere, or will it autocreate if it doesn't exist already?

Reply:
The queue name is a column value in the `graphile_worker` database, so you don't need to define it anywhere.